docs/section-metadata.md

```diff
@@ -231,21 +231,27 @@ Narrow style puts the content in a section that is fixed to **610px**.
 Adds a divider before or after the section.
```

Comment:
Description probably needs to be updated as well, as there does not seem to be an option for `before` and `after` anymore.